### ISSUE TITLE
delete g.regeneratorRuntime

### ISSUE THREAD

**Issue by AutoSponge** [babel/babel#1412]
I've tried to track this down but I can't find the source to patch. Not sure if it's in babel or facebook's regenerator project.

I suggest changing this:

```js
var hadRuntime = g.regeneratorRuntime && Object.getOwnPropertyNames(g).indexOf("regeneratorRuntime") >= 0;
var oldRuntime = hadRuntime && g.regeneratorRuntime;
delete g.regeneratorRuntime;
```

To this:

```js
var hadRuntime = g.regeneratorRuntime && Object.getOwnPropertyNames(g).indexOf("regeneratorRuntime") >= 0;
var oldRuntime = hadRuntime && g.regeneratorRuntime;
try {
  delete g.regeneratorRuntime;
} catch (e) {
  g.regeneratorRuntime = undefined;
}
```

It doesn't fail silently now in IE but this will.

**Comment by monsanto** [comment]
Can you please take this up with Regenerator upstream?

**Comment by AutoSponge** [comment]
track https://github.com/facebook/regenerator/issues/195

**Comment by sebmck** [comment]
Closed via facebook/regenerator#196.
